api/app/composer.json

```diff
@@ -27,7 +27,7 @@
     "require": {
         "php": "^8.2",
         "aws/aws-sdk-php": "3.329.0",
-        "doctrine/doctrine-bundle": "^2.13",
+        "doctrine/doctrine-bundle": "2.13.3",
```

Comment:
Are we pinning this for a specific reason?

Reply:
Yes we have added a reminder to unpin this but we had to for now to get it working as latest version doesn't work properly with other upgrade.